src/SampSharp.GameMode/World/VehicleModelInfo.cs

```diff
@@ -267,6 +269,11 @@ private VehicleModelInfo(int type, string name, VehicleCategory category)
         /// </summary>
         public VehicleCategory Category { get; private set; }
 
+        /// <summary>
+        ///     Gets the seats number of this <see cref="VehicleModelInfo" />.
+        /// </summary>
+        public int SeatsNumber { get; private set; }
```

Comment:
Could you rename this to `SeatCount` along with the parameter in the constructor ?

Reply:
Done